collapsiblecalendarview2/src/main/res/layout/widget_collapsible_calendarview.xml

```diff
@@ -138,7 +137,7 @@
         <RelativeLayout
             android:id="@+id/frame_today_icon"
             android:layout_width="wrap_content"
-            android:layout_height="match_parent"
+            android:layout_height="0dp"
```

Comment:
What is this supposed to do?

Reply:
0dp with the constraints it has makes the button height as tall as the parent view